src/structures/Presence.js

```diff
@@ -104,14 +98,14 @@ class Presence {
      * @property {?ClientPresenceStatus} mobile The current presence in the mobile application
      * @property {?ClientPresenceStatus} desktop The current presence in the desktop application
      */
-    this.clientStatus = data.client_status || null;
+    this.clientStatus = data.client_status ?? null;
 
     return this;
   }
 
   _clone() {
     const clone = Object.assign(Object.create(this), this);
-    if (this.activities) clone.activities = this.activities.map(activity => activity._clone());
+    clone.activities = this.activities.map(activity => activity._clone());
```

Comment:
```suggestion
    clone.activities = this.activities?.map(activity => activity._clone());
```

Reply:
i don't think `this.activities` can ever be undefined